cpp/src/interop/from_arrow_host_strings.cu

```diff
@@ -152,8 +152,10 @@ std::unique_ptr<column> from_arrow_stringview(ArrowSchemaView* schema,
   // first copy stringview array to device
   auto items   = view.buffer_views[stringview_vector_idx].data.as_binary_view;
   auto d_items = rmm::device_uvector<ArrowBinaryView>(input->length, stream, mr);
+  // offset should not be > input->length which caller insures is < max size_type
+  auto const offset = static_cast<cudf::size_type>(input->offset);
```

Comment:
```suggestion
  // offset should not be > input->length which caller ensures is < max size_type
  auto const offset = static_cast<cudf::size_type>(input->offset);
```

Is this comment correct? Why must offset be `<= length`? e.g. Consider:

```
import pyarrow as pa
import numpy as np
col = pa.array(np.zeros(100, dtype="int8"))

sliced = col.slice(50, 2)
```
`input->length` will be `2`, but `input->offset` will be `50`. The same applies mutatis mutandis for different dtypes.